api-reference/10 UI Widgets/dxGantt/1 Configuration/resourceAssignments/dataSource.md

```diff
@@ -65,37 +57,22 @@ Binds the widget to the data source, which contains resource assignments.
 
     <!-- tab: App.vue -->
     <template>
-        <DxGantt ...
-            <DxResourceAssignments :data-source="resourceAssignmentsDataSource"/>
-			// ...
+        <DxGantt ... >
+            <DxResourceAssignments :data-source="resourceAssignmentsDataSource" />
+            <!-- ... -->
         </DxGantt>
     </template>
     <script>
-        import {
-            DxGantt,
-            DxResourceAssignments,
-			// ...
-        } from 'devextreme-vue/gantt';
-
-        import {
-            resourceAssignments,
-			// ...
-        } from './data.js';
-
+        import { DxGantt, DxResourceAssignments, ... } from 'devextreme-vue/gantt';
+        import { resourceAssignments, ... } from './data.js';
         export default {
-            components: {
-                DxResourceAssignments,
-				// ...
-            },
+            components: { DxResourceAssignments, ... },
             data() {
-                return {
-                resourceAssignments,
-				// ...
-                };
+                return { resourceAssignments, ... };
             },
             computed: {
                 resourceAssignmentsDataSource() { return resourceAssignments; },
```

Comment:
You already return the `resourceAssignments` array from the `data()` function. What do you need the computed property for?